scripts/cnode-helper-scripts/blockLog.sh

```diff
@@ -45,7 +45,7 @@ getDeltaMS() {
 while true
 do
   blockHeight=$(curl -s -H 'Accept: application/json' http://${EKG_HOST}:${EKG_PORT}/ | jq -r .cardano.node.metrics.blockNum.int.val)
-  if [ -z $blockHeight ] || [ "$blockHeight" == "null" ] || [ "$blockHeightPrev" == "null" ] ; then
+  if [ -z $blockHeight ] || [ "$blockHeight" == "null" ]; then
     echo "WARN: can't query EKG on http://127.0.0.1:$(cat $CONFIG | jq .hasEKG)/ ..."
```

Comment:
This error message should also use ${EKG_PORT} ?

Reply:
ok, done